src/pageComponents/dashboard/index.tsx

```diff
@@ -1058,12 +1059,16 @@ const Dashboard = () => {
 										"Only collection owner can delete this collection",
 									);
 								}
+
+								if (isDataPresentCheck) {
```

Comment:
can we add the `current` in line 1069 in this condition ? the current tells `only push to home if user is deleting the category when user is currently in that category` , if we dont add this then we might be missing on this functionality